libs/martech/martech.js

```diff
@@ -118,14 +118,14 @@ export const getTargetPersonalization = async () => {
   const responseStart = Date.now();
   window.addEventListener(ALLOY_SEND_EVENT, () => {
     const responseTime = calculateResponseTime(responseStart);
-    window.lana.log(`target response time: ${responseTime}`, { tags: 'errorType=info,module=martech' });
+    window.lana.log(`target response time: ${responseTime}`, { tags: 'module-martech', errorType: 'i' });
   }, { once: true });
 
   let manifests = [];
   let propositions = [];
   const response = await waitForEventOrTimeout(ALLOY_SEND_EVENT, timeout);
   if (response.error) {
-    window.lana.log('target response time: ad blocker', { tags: 'errorType=info,module=martech' });
+    window.lana.log('target response time: ad blocker', { tags: 'errorType-info,module-martech' });
```

Comment:
isn't this not consistent with the above? shouldn't it be 
`{ tags: 'module-martech', errorType: 'i' }` too?

Reply:
It should be, if the log was informational but since this is clearly inside the error block i have left the errorType to default which is "error".

Reply:
mm to me "error" is a problem on our side, the fact the browser uses adblocker does not really fall into that category, but i guess i'm fine :)